src/ludamus/templates/chronology/session/edit.html

```
@@ -0,0 +1,198 @@
+                                                id="id_sf_{{ field.slug }}"
+                                                class="w-full border border-border rounded-lg px-4 py-2 focus:outline-none focus:ring-2 focus:ring-primary">
+                                            <option value="">—</option>
+                                            {% for option in field.options %}
+                                                <option value="{{ option.value }}"
+                                                        {% if current_value == option.value %}selected{% endif %}>
+                                                    {{ option.label }}
+                                                </option>
+                                            {% endfor %}
+                                        </select>
+                                        {% if field.allow_custom %}
+                                            <input type="text"
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)